tests/minimal_webgl/CMakeLists.txt

```diff
@@ -78,6 +80,11 @@ if (EMSCRIPTEN)
 
 	# Reduce code size: We do not need native POSIX filesystem emulation support (Emscripten FS/MEMFS)
 	append_linker_flags("-s FILESYSTEM=0")
+
+	if(MEMORY64)
+		append_linker_flags("-s MEMORY64=1")
+		add_definitions("-s MEMORY64=1")
+	endif()
```

Comment:
Can you just do "`CMAKE_LINK_FLAGS="-sMEMORY64"` generically to avoid modifying the makefile  (not that you can drop the space and the `=1` these days).